app/globals.css

```diff
@@ -83,3 +83,35 @@ html.dark {
   --background: var(--dark-background);
   --foreground: var(--dark-foreground);
 }
+
+@keyframes circular-motion {
+  0%, 100% { 
+    box-shadow: 8px 0 12px -3px, 3px 0 5px -3px;
+  }
+  12.5% { 
+    box-shadow: 5.6px 5.6px 12px -3px, 2.1px 2.1px 5px -3px;
+  }
+  25% { 
+    box-shadow: 0 8px 12px -3px, 0 3px 5px -3px;
+  }
+  37.5% { 
+    box-shadow: -5.6px 5.6px 12px -3px, -2.1px 2.1px 5px -3px;
+  }
+  50% { 
+    box-shadow: -8px 0 12px -3px, -3px 0 5px -3px;
+  }
+  62.5% { 
+    box-shadow: -5.6px -5.6px 12px -3px, -2.1px -2.1px 5px -3px;
+  }
+  75% { 
+    box-shadow: 0 -8px 12px -3px, 0 -3px 5px -3px;
+  }
+  87.5% { 
+    box-shadow: 5.6px -5.6px 12px -3px, 2.1px -2.1px 5px -3px;
+  }
+}
+
+
+.circular-shadow-motion {
+  animation: circular-motion 8s linear infinite;
+}
```

Comment:
why not move this inside the tailwind config https://tailwindcss.com/docs/animation